docs/api-docs/analytics/bodl-overview.md

```diff
@@ -0,0 +1,423 @@
+# Big Open Data Layer (BODL) 
+
+## Overview
+
+The Big Open Data Layer (shortened as `BODL`, pronounced 'Bottle') is a JavaScript object that exposes storefront data points to BigCommerce and third-party analytics integrations. Instantiating a standard `BODL` object on a BigCommerce-hosted or headless storefront will make the site faster and provide richer, more consistent analytics.
```

Comment:
"Instantiating a standard `BODL` object on a BigCommerce-hosted or headless storefront will make the site faster and provide richer, more consistent analytics." — this statement isn't true as written. 

Loading the BODL doesn't actually make the site faster.

The benefit comes into play in the scenario where a merchant has 2 or more analytics scripts, instead of each one fetching data independently, if the 2 or more scripts get their data from the BODL and the BODL is the only thing that fetches data from our system, then that's a more efficient way to get and distribute data and theoretically makes things faster because we're fetching data once just for the BODL rather than once per script.

Reply:
@bc-derrekpearson great, we'll incorporate this into the next draft.

🎉  on your first GitHub comment!